django_project/settings.py

```diff
@@ -151,3 +151,12 @@
 
 # Minimal string length considered similarity
 MIN_SIMILARITY_LENGTH = 10
+
+# Email settings, service provider SendGrid
+SENDGRID_API_KEY = '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'
```

Comment:
I don't think this type of variable should be exposed in public repository. 
Please:
- rotate this key
- save it all into .env file please. 

Please, never expose such credentials ever again.